staging/src/k8s.io/client-go/tools/clientcmd/api/helpers.go

```diff
@@ -189,3 +191,79 @@ func MakeAbs(path, base string) (string, error) {
 	}
 	return filepath.Join(base, path), nil
 }
+
+// RedactSecrets replaces any sensitive values with REDACTED
+func RedactSecrets(config *Config) error {
+	if err := redactSecrets(reflect.ValueOf(config), false); err != nil {
```

Comment:
Make it simpler :wink:, you don't need the if at all:
```golang
	return redactSecrets(reflect.ValueOf(config), false)
```

Reply:
Doh, thank you.